lms/static/js/student_profile/views/learner_profile_fields.js

```diff
@@ -16,22 +17,31 @@
             },
 
             showNotificationMessage: function () {
-                var accountSettingsLink = '<a href="' + this.options.accountSettingsPageUrl + '">' + gettext('Account Settings page.') + '</a>';
+                var accountSettingsLink = HtmlUtils.joinHtml(
+                    HtmlUtils.interpolateHtml(
+                        HtmlUtils.HTML('<a href="{settings_url}">', {settings_url: this.options.accountSettingsPageUrl})
+                    ),
```

Comment:
I think the brackets are in the wrong place, as the interpolation parameters are incorrectly being passed to ``HtmlUtils.HTML``.